server.js

```diff
@@ -1,29 +1,192 @@
-import express from 'express'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from 'express';
+import cors from 'cors';
+import mongoose from 'mongoose';
+import listEndpoints from 'express-list-endpoints';
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
+const mongoUrl = process.env.MONGO_URL || 'mongodb://localhost/happyThoughts';
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
+
+//Schema for a more clean and dynamic approach
+const ThoughtSchema = new mongoose.Schema({
+	message: {
+		type: String,
+		minlength: 5,
+		maxlength: 140,
+		trim: true,
+		required: true,
+	},
+	// categories: {
+	// 	type: String,
+	// 	enum: [
+	// 		'Food thought',
+	// 		'Project thought',
+	// 		'Meditation thought',
+	// 		'Travel thought',
+	// 		'Other thought',
+	// 	],
+	// 	required: true,
+	// },
+	hearts: {
+		type: Number,
+		default: 0,
+	},
+	createdAt: {
+		type: Number,
+		default: () => Date.now(),
+	},
+	author: {
+		type: String,
+		default: 'Anonymous',
+	},
+});
+
+const Thought = mongoose.model('Thought', ThoughtSchema);
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(express.json())
+app.use(cors());
+app.use(express.json());
 
+//----ENDPOINTS------
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+	res.send(
+		'Hello world, Welcome to Madelene Happy Thoughts API  - see this API live at 👉:https://mt-dotse-happy-thoughts.netlify.app/'
```

Comment:
Very clever adding your netlify link in the response, so it's easier to check it!